package.json

```
@@ -2,10 +2,11 @@
 	"name": "api-demo",
 	"version": "1.0.0",
 	"description": "Tasks API",
-	"main": "server.js",
+	"main": "src/server.js",
 	"scripts": {
-		"start": "nodemon src/server.js",
+		"start": "nodemon src/server.ts",
```

If you change this to `"start": "node ./build/server.js",` the command `npm start` should work. 
```suggestion
		"start": "nodemon ./build/server.js",
```